### PR TITLE
Added member profile for Vaibhav bisht

### LINKED ISSUE #4438: [Community] Member Profile: Vaibhav
**Current Behavior**

@vaibhav135 has been a consistent contributor and community member.

**Desired Situation**
Let's recognize @vaibhav135 as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: @vaibhav135
- Twitter: [Vaibhav__Bisht](https://twitter.com/Vaibhav__Bisht)
- LinkedIn: [vaibhav-singh-bisht-81619b126](https://www.linkedin.com/in/vaibhav-singh-bisht-81619b126/)
- Meshery Cloud: [b556e60f-f560-4f3c-9a74-ea0327f49163](https://meshery.layer5.io/user/b556e60f-f560-4f3c-9a74-ea0327f49163)
- Link to profile picture:

A detailed explanation on how to set up a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](https://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by DarrenDsouza7273**
**Description**
Added the member profile for Vaibhav 
This PR fixes #4438 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit eaf696afe7a46aeff85f19f1642326dcb40a027b at: https://649ae46ca0fd760a573e99a4--layer5.netlify.app

**Comment by DarrenDsouza7273** [comment]
@vaibhav135

**Comment by DarrenDsouza7273** [comment]
@Chadha93

**Comment by l5io** [comment]
🚀 Preview for commit 1856d93cf63c5b34a222e791a9460a284ada2e65 at: https://649be0d4617c61443031df89--layer5.netlify.app

File: src/collections/members/vaibhav-bisht/index.mdx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+name: Vaibhav Bisht
+position: Contributor
+image_path: ./vaibhav-bisht.webp
+github: vaibhav135
+twitter: Vaibhav_bisht
```
Comment (hirentimbadiya) [inline src/collections/members/vaibhav-bisht/index.mdx:6]: Vaibhav_Bisht

**Review by hirentimbadiya — CHANGES_REQUESTED** [review]
Change the twitter username with the correct one.

**Comment by vaibhav135** [comment]
@DarrenDsouza7273 Yes LGTM 🚀

**Comment by DarrenDsouza7273** [comment]
@hirentimbadiya done

**Comment by l5io** [comment]
🚀 Preview for commit 63ee533c9234f668ee5ed6960c546298e87d3d80 at: https://649c3b772402b80d2362fa4d--layer5.netlify.app

**Review by hirentimbadiya — APPROVED** [review]
LGTM ✅

**Comment by l5io** [comment]
🚀 Preview for commit 6a59e38a974d9eb9f04ea3232632a8d3c05d3d0c at: https://649d2a603b49663d39127c86--layer5.netlify.app

**Comment by goldfishdolphin** [comment]
Looking good 👍🏽

**Review by Chadha93 — APPROVED** [review]
LGTM

**Comment by Chadha93** [comment]
Yay @vaibhav135 :balloon:

**Pull request merged**